.github/workflows/release.yml

```diff
@@ -29,7 +29,7 @@ jobs:
           BUILD_TYPE="tarball"
           FLAKE="github:${GITHUB_REPOSITORY}/${GITHUB_REF_NAME}"
 
-          nix build -L ${FLAKE}#chain-maind-$BUILD_TYPE
+          nix build -L ${FLAKE}#chain-maind-testnet-$BUILD_TYPE
           cp result chain-main_${GITHUB_REF_NAME:1}_${PLATFORM}.tar.gz
```

Comment:
is this correct?

Reply:
Yes. The DB backend defaults to `goleveldb`.

Reply:
The asset name should has the surfix too: `chain-main_4.0.0-alpha3-croeseid_Darwin_arm64_croeseid.tar.gz`

Reply:
@allthatjazzleo Can you check the final asset name? I remember you removed the `_croeseid` postfix in https://github.com/crypto-org-chain/chain-main/pull/776 and all the automation scripts assume that there's no `_croeseid` postfix.

Reply:
@yihuang @devashishdxt 
according to our previous convention, we don't have `testnet` suffix
https://github.com/crypto-org-chain/chain-main/releases/tag/v3.1.1-croeseid

Reply:
@allthatjazzleo Final generated assets will look like this: https://github.com/devashishdxt/chain-main/releases/tag/untagged-cab15c548893a1949350

Reply:
> @allthatjazzleo Can you check the final asset name? I remember you removed the `_croeseid` postfix in #776 and all the automation scripts assume that there's no `_croeseid` postfix.

yes coz croeseid is already part of the version tag